### PR TITLE
Moved core programme material info to a public space

### PR DESCRIPTION
**Pull request by BroiSatse**
### Context
https://trello.com/c/6uM6MLRA/596-move-the-cip-guidance-to-a-new-publicly-accessible-page

### Changes proposed in this pull request

The core material info page has been moved to a public pages controller. Original page has been replaced with simpler page linking to the new public page.

### CODE REVIEW

**Comment by BroiSatse** [comment]
[Move the CIP guidance to a new, publicly accessible page](https://trello.com/c/6uM6MLRA/596-move-the-cip-guidance-to-a-new-publicly-accessible-page)

File: app/views/pages/core_materials_info.html.erb [hunk 1]
```diff
@@ -0,0 +1,225 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
```
Comment (ethanmills) [inline app/views/pages/core_materials_info.html.erb:1]: Do we have a page title here?

File: app/views/schools/core_programme/materials/info.html.erb [hunk 2]
```diff
@@ -1,228 +1,9 @@
 <% content_for :before_content, govuk_back_link( text: "Back", href: schools_cohort_path(@cohort.start_year)) %>
```
Comment (ethanmills) [inline app/views/schools/core_programme/materials/info.html.erb:1]: Would you be able to check this group of pages for page titles please? I noticed some missing when looking at google analytics

File: app/views/schools/core_programme/materials/info.html.erb [hunk 3]
```diff
@@ -1,228 +1,9 @@
...
 
 <span class="govuk-caption-xl"><%= @cohort.start_year %> cohort</span>
-<h1 class="govuk-heading-xl">Choose study materials for your early career teachers</h1>
+<h1 class="govuk-heading-xl">Do you know which accredited materials you want to use?</h1>
```
[the remark this answers is not in the document]
Comment (ethanmills) [reply app/views/schools/core_programme/materials/info.html.erb:4]: Looks like it didn't get merged: https://github.com/DFE-Digital/early-careers-framework/pull/322/files
Would you mind just double checking there's nothing in there that we're missing here, then closing that PR please?

File: app/controllers/lead_providers/partnership_csv_uploads_controller.rb [hunk 4]
```diff
@@ -10,7 +10,7 @@ def create
       # TODO: do we want to catch empty session[:delivery_partner_id] ?
       if params[:partnership_csv_upload].blank?
         @partnership_csv_upload = PartnershipCsvUpload.new
-        @partnership_csv_upload.errors[:base] << "Please select a CSV file to upload."
+        @partnership_csv_upload.errors.add(:base, "Please select a CSV file to upload.")
```
Comment (kicferk1) [inline app/controllers/lead_providers/partnership_csv_uploads_controller.rb:13]: Nice

File: app/views/schools/core_programme/materials/info.html.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (BroiSatse) [reply app/views/schools/core_programme/materials/info.html.erb:4]: @ethanmills Checked - both contents are the same. :)

File: app/views/pages/core_materials_info.html.erb (hunk 1 above)
Comment (BroiSatse) [reply app/views/pages/core_materials_info.html.erb:1]: There is no page title in the content file. I'll think of some :)

File: config/routes.rb [hunk 5]
```diff
@@ -10,7 +10,7 @@
     post "/users/sign_in_with_token", to: "users/sessions#sign_in_with_token"
   end
 
-  get "/pages/:page", to: "pages#show"
+  get "/pages/:page", to: "pages#show", as: :page
```
[the remark this answers is not in the document]
Comment (BroiSatse) [reply config/routes.rb:13]: Added :)

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**